build/tc470422/application-template-stibuild.json

```diff
@@ -137,7 +137,7 @@
           "sourceStrategy": {
             "from": {
               "kind": "ImageStreamTag",
-              "name": "ruby-22-centos7:latest"
+              "name": "ruby-25-centos7:latest"
```

Comment:
Please change to use openshift imagestream like above, if you use this image directly, the case will be failed in disconnected cluster; but if use openshift imagestream, we can mirror/import ruby image.